### PR TITLE
Add timing information for greedy

### PR DESCRIPTION
**Pull request by kevinchern**
The PR for timing for SA for reference: https://github.com/dwavesystems/dwave-samplers/pull/54/

### CODE REVIEW

File: tests/test_greedy_sampler.py [hunk 1]
```diff
@@ -280,3 +280,36 @@ def test_initial_states_sample_like(self, initial_states):
...
+            self.sample_sets.append(sample_set)
+
+        self.timing_keys = ["preprocessing_ns", "postprocessing_ns", "sampling_ns"]
+
+    def test_keys_exist(self):
+        for sample_set in self.sample_sets:
+            self.assertIn("timing", sample_set.info)
+            timing = sample_set.info['timing']
+            for timing_key in self.timing_keys:
+                self.assertIn(timing_key, timing)
```
Comment (randomir) [inline tests/test_greedy_sampler.py:306]: You could use `issubset` here, and mark subtests:
```suggestion
        self.timing_keys = set(["preprocessing_ns", "postprocessing_ns", "sampling_ns"])

    def test_keys_exist(self):
        for sample_set in self.sample_sets:
            with self.subTest(ss=sample_set):
                self.timing_keys.issubset(sample_set.info['timing'])
```

**Review by randomir — APPROVED** [review]
LGTM!

File: tests/test_greedy_sampler.py (hunk 1 above)
Comment (kevinchern) [reply tests/test_greedy_sampler.py]: Thanks, that's much more concise! Can I make the same changes for simulated annealing's test in *this* PR?

Comment (randomir) [reply tests/test_greedy_sampler.py]: Sure, why not.

**Pull request merged**